### PR TITLE
Fixed bug with Questa test_args parameters converting

### PR DESCRIPTION
**Pull request by vborchsh**
There is a simple bug: in case of spaces in the test_args (e.g. -L xpm) they would be converted into something like `-L\ xpm` and so on. Which is unacceptable from vsim.

There is a working example in the past cocotb_test: https://github.com/themperek/cocotb-test/blob/32b19b0813ae40cefc2b6d5bf1a1a715682fb18c/cocotb_test/simulator.py#LL615C43-L615C43

### CODE REVIEW

**Comment by themperek** [comment]
So the intention would be `test_args` to be a list `["-L", "xpm"]`.

I agree it is a bit inconvenient and we 
- call it `split()`
- do as the PR proposes?

Any opinion?

**Comment by vborchsh** [comment]
> So the intention would be test_args to be a list ["-L", "xpm"].

Btw, looks acceptable from my point. Didn't think this way. But I'd like test it first :)

**Comment by themperek** [comment]
@vborchsh In case this is ok would be great to add few word into documentation about this.

**Comment by vborchsh** [comment]
Ok, let's leave it as is.

**Comment by ktbarrett** [comment]
@themperek Are we good?

File: cocotb/runner.py [hunk 1]
```diff
@@ -226,7 +226,8 @@ def test(
                 If not set, run all testcases found in *test_module*.
                 Can be a comma-separated list.
             seed: A specific random seed to use.
-            test_args: Extra arguments for the simulator.
+            test_args: Extra arguments for the simulator. Mustn't include space symbols.
+                In case of space split command into list.
```
Comment (themperek) [inline cocotb/runner.py:230]:
```suggestion
            test_args: A list of extra arguments for the simulator.
```
Like this?

**Review by themperek — APPROVED** [review]

**Comment by imphil** [comment]
Argh, didn't look at the commit structure and didn't do a squash :(

@vborchsh Please try to fold changes into a clean commit structure, i.e. a single commit with a proper commit message. You can use (on your local machine) git's interactive rebase for this. (No harm done this time, though, apart from a less nice git history.)

**Pull request merged**